src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -443,11 +439,14 @@ public function close()
 	 */
 	public function __construct(array $options)
 	{
-        if (isset($options['log4php.properties'])) {
-            \Logger::configure($options['log4php.properties']);
-        }
-        $this->_logger = \Logger::getLogger('Pami.ClientImpl');
-	    $this->_host = $options['host'];
+		$this->setLogger(
+			new Log4phpAdapter(
```

Comment:
I'm not sure about hardcoding this here. Either we should be injecting the logger by constructor or just leave things as they are, since the logger remains completely internal to pami. Or maybe I'm missing something?